docker-compose-test.yml

```diff
@@ -14,8 +16,8 @@ services:
     image: rodolpheche/wiremock
     networks:
     - hmpps_int
```

Comment:
container name needs to be unique,
also unless-stopped means multiple project wont try to start when you start docker desktop